### PR TITLE
MAINT: Use PyArray_ISBYTESWAPPED instead of !PyArray_ISNOTSWAPPED.

### PR DESCRIPTION
**Pull request by jaimefrio**
Positives are easier to parse than double negatives.

### CODE REVIEW

File: numpy/core/src/multiarray/arraytypes.c.src [hunk 1]
```diff
@@ -2459,13 +2463,13 @@ UNICODE_nonzero (npy_ucs4 *ip, PyArrayObject *ap)
     npy_bool seen_null = NPY_FALSE;
     char *buffer = NULL;
 
-    if ((!PyArray_ISNOTSWAPPED(ap)) || (!PyArray_ISALIGNED(ap))) {
+    if ((PyArray_ISBYTESWAPPED(ap)) || (!PyArray_ISALIGNED(ap))) {
```
Comment (eric-wieser) [inline numpy/core/src/multiarray/arraytypes.c.src]: Parens have gone from unnecessary to very unnecessary here

**Review by eric-wieser — APPROVED** [review]
LGTM

File: numpy/core/src/multiarray/arraytypes.c.src (hunk 1 above)
Comment (jaimefrio) [inline numpy/core/src/multiarray/arraytypes.c.src]: Indeed, have removed them all in the last push.

**Comment by eric-wieser** [comment]
Thanks @jaimefrio

**Pull request merged**